.github/workflows/build_orca.yml

```
@@ -217,7 +217,7 @@ jobs:
           fi
 
       - name: Create DMG without notary
-        if: github.ref != 'refs/heads/main' && runner.os == 'macOS' && inputs.macos-combine-only
+        if: !((github.repository == 'OrcaSlicer/OrcaSlicer' && (github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/heads/release/'))) || (vars.ENABLE_SIGNING == 'true' && github.event_name != 'pull_request')) && runner.os == 'macOS' && inputs.macos-combine-only
```

actionlint (1.7.11)

[error] unexpected token "==" while parsing variable access, function call, null, bool, int, float or string. expecting "IDENT", "(", "INTEGER", "FLOAT", "STRING"

(expression)